run-integration-tests.py

```diff
@@ -75,8 +76,8 @@ def run_python_integration_tests(root_dir, version, test_name, extra_maven_repo,
         package += "," + extra_packages
 
     jars = jars and ["--jars", jars] or []
+    repos = extra_maven_repo and ["--repositories", extra_maven_repo] or []
```

Comment:
While this works, I feel like it's very unclear to the reader.

Even `["--repositories", extra_maven_repo] if extra_maven_repo else []` I think would be clearer. But good to merge? And I can adjust since I'll be messing with some of this code a little bit anyways.